enginetest/queries/script_queries.go

```diff
@@ -2322,6 +2322,18 @@ var ScriptTests = []ScriptTest{
 			},
 		},
 	},
+	{
+		Name: "can't create view with same name as existing table",
```

Comment:
Also check the opposite: creating a table with the same name as an existing view